sources/packages/backend/apps/api/src/services/institution/institution.service.ts

```diff
@@ -761,6 +761,65 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     return note;
   }
 
+  /**
+   * Service to add note for an Institution.
+   * @param institutionId
+   * @param noteType note type.
+   * @param noteDescription note description.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @returns saved Note.
+   */
+  async addInstitutionNote(
```

Comment:
why do we need this service? we already had this feature right.

Reply:
The other method is the `saveInstitutionNote`, and It is still used in other places, right?
Are the other places failing also?